scalastyle-config.xml

```diff
@@ -268,10 +268,7 @@ This file is divided into 3 sections:
   </check>
 
   <!-- SPARK-16877: Avoid Java annotations -->
-  <check customId="OverrideJavaCase" level="error" class="org.scalastyle.scalariform.TokenChecker" enabled="true">
-    <parameters><parameter name="regex">^Override$</parameter></parameters>
-    <customMessage>override modifier should be used instead of @java.lang.Override.</customMessage>
-  </check>
+  <check level="error" class="org.scalastyle.scalariform.OverrideJavaChecker" enabled="true"></check>
```

Comment:
This removes the workaround we used and use `org.scalastyle.scalariform.OverrideJavaChecker` -  `https://github.com/scalastyle/scalastyle/issues/214`.

I manually tested with `@Override`:

Before:

```
[error] ...:435:3: override modifier should be used instead of @java.lang.Override.
```

After:


```
[error] ...:435:2: override keyword should be used instead of @java.lang.Override
```